modules/reporting/app/controllers/cost_reports_controller.rb

```diff
@@ -76,23 +76,38 @@ def check_cache
 
   def index
     table
+    return if performed?
 
-    unless performed?
-      respond_to do |format|
-        format.html do
-          session[report_engine.name.underscore.to_sym].try(:delete, :name)
-          render locals: { menu_name: project_or_global_menu }
-        end
+    respond_to do |format|
+      format.html { render_html }
+      format.xls { export(:xls) }
+      format.pdf { export(:pdf) }
```

Comment:
Question is here, we're making the timesheet "the" representation of the cost export as a PDF. WDYT about adding a second parameter like `variant` or something where we specify that we want the cost export or the timesheet. In a later iteration we could even make this a parameter

Reply:
Yes, true. This should and will be a parameter when a cost report PDF is implemented.